nextcord/components.py

```diff
@@ -268,6 +275,12 @@ def to_dict(self) -> SelectMenuPayload:
         if self.placeholder:
             payload["placeholder"] = self.placeholder
 
+        if hasattr(self, "channel_types"):
```

Comment:
resolved, is still an empty list because people might not want to check if .channel_types == None, people expect it to be a list instead of Optional[list]